cmd/banner.go

```diff
@@ -69,9 +66,35 @@ var bannerCmd = &cobra.Command{
 		fmt.Print(haBanner)
 		fmt.Println()
 
-		fmt.Println("System information")
+		nowait, err := cmd.Flags().GetBool("no-wait-for-supervisor")
+		if err != nil {
+			fmt.Println(err)
+			ExitWithError = true
+		}
+
+		if !nowait {
+			for i := 0; i < 60; i++ {
+				// We could use ping here, but Supervisor loads networking data asynchronously.
+				// Networking info are very useful, so wait until networking data have been
+				// loaded...
+				netinfo, err := supervisorGet("network", "info")
```

Comment:
Use the ping endpoint

Reply:
Using ping leads to no network info (see comment above the code):

```
| |  | |                          /\           (_)   | |            | |  
| |__| | ___  _ __ ___   ___     /  \   ___ ___ _ ___| |_ __ _ _ __ | |_ 
|  __  |/ _ \| '_ \ _ \ / _ \   / /\ \ / __/ __| / __| __/ _\ | '_ \| __|
| |  | | (_) | | | | | |  __/  / ____ \\__ \__ \ \__ \ || (_| | | | | |_ 
|_|  |_|\___/|_| |_| |_|\___| /_/    \_\___/___/_|___/\__\__,_|_| |_|\__|

Welcome to the Home Assistant command line.

Waiting for Supervisor to startup...
System information

  OS Version:               Home Assistant OS 6.0.dev20210518
  Home Assistant Core:      2021.5.4

  Home Assistant URL:       http://homeassistant.local:8123
  Observer URL:             http://homeassistant.local:4357
```

Reply:
https://github.com/home-assistant/supervisor/blob/main/supervisor/api/middleware_security.py#L122 this block all API request. host/info is in the same second valid as supervisor/ping